### DISCUSSION TITLE
Frage zum Button & Jalousie Element

### DISCUSSION THREAD

**Discussion by lukas3482** [Zefau/ioBroker.jarvis#919] [Q&A] [answered]
Hi,

ich hätte eine Frage zum Button Element & zum Jalousie Element.

1. Button Element
Ich würde gern einen "Zentral Aus" Button in meiner Visu integrieren.
![](https://i.imgur.com/ocH3san.png)
Meine Frage wäre ob es möglich ist den Wert wenn der Button gedrückt wird der an das Datenobjekt gesendet wird zu bearbeiten. (Also in diesem Fall soll immer wenn der Button gedrückt wird "false" gesendet werden)

2. Jalousie Element
![](https://i.imgur.com/rItpoKc.png)
Die Funktionen dieser beiden Buttons sind bei mir vertauscht. -> Wenn ich den Pfeil nach oben drücke fährt meine Jalousie nach unten und umgekehrt.

Jarvis Version: 2.2.3

Mfg

Lukas

**Comment by Zefau** [comment, marked as answer]
>  Meine Frage wäre ob es möglich ist den Wert wenn der Button gedrückt wird der an das Datenobjekt gesendet wird zu bearbeiten. (Also in diesem Fall soll immer wenn der Button gedrückt wird "false" gesendet werden)

Dies musst du (aktuell noch) über einen eigenen Datenpunkt in ioBroker via Blockly lösen.

>  Die Funktionen dieser beiden Buttons sind bei mir vertauscht. -> Wenn ich den Pfeil nach oben drücke fährt meine Jalousie nach unten und umgekehrt.

Das geht mittels Datenpunkt Eigenschaften, siehe https://zefau.gitbook.io/jarvis-v2/geraete/devices#datenpunkt-eigenschaften
Einfach `min` auf `100` und `max` auf `0` setzen.

**Reply by lukas3482** [reply]
Funktioniert. Danke für die schnelle Antwort!
